reports-scheduler/src/main/kotlin/org/opensearch/reportsscheduler/ReportsSchedulerPlugin.kt

```diff
@@ -80,15 +80,15 @@ import org.opensearch.watcher.ResourceWatcherService
 import java.util.function.Supplier
 
 /**
- * Entry point of the OpenDistro for OpenSearch Reports scheduler plugin.
+ * Entry point of the OpenSearch Reports scheduler plugin.
  * This class initializes the rest handlers.
  */
 class ReportsSchedulerPlugin : Plugin(), ActionPlugin, JobSchedulerExtension {
 
     companion object {
-        const val PLUGIN_NAME = "opendistro-reports-scheduler"
+        const val PLUGIN_NAME = "opensearch-reports-scheduler"
         const val LOG_PREFIX = "reports"
-        const val BASE_REPORTS_URI = "/_opendistro/_reports"
+        const val BASE_REPORTS_URI = "/_plugins/_reports"
```

Comment:
does this remove the `/_opendistro/_reports` API? I thought we still need to be compatible with them?